build.sh

```diff
@@ -46,6 +46,9 @@ elif [ "$target" == "mkrelease" ]; then
     $boarddir/mkimage.sh
     cp $outputdir/images/$osname-$board.img $basedir
     mv $basedir/$osname-$board.img  $basedir/$osname-$board-$osversion.img
+    rm -f $basedir/$osname-$board-$osversion.img.xz
+    xz -6ek $basedir/$osname-$board-$osversion.img
+    echo "your image is ready at $basedir/$osname-$board-$osversion.img.xz"
```

Comment:
Just a minor remark: it would be nice if the script said "your xz image is ready at..." and then, one line later, "your gz image is ready at...".

Reply:
I've updated the print message.